src/execution/operator/persistent/buffered_csv_reader.cpp

```diff
@@ -1772,6 +1796,31 @@ void BufferedCSVReader::Flush(DataChunk &insert_chunk) {
 			}
 		}
 	}
+	if (conversion_error_ignored && options.ignore_errors) {
+		SelectionVector succesful_rows;
```

Comment:
This should only be triggered if `options.ignore_errors` is true so the second condition can be turned into an assertion